app/src/lib/app-state.ts

```diff
@@ -167,7 +168,7 @@ export type Popup =
       repository: Repository
       files: ReadonlyArray<WorkingDirectoryFileChange>
     }
-  | { type: PopupType.Preferences }
+  | { type: PopupType.Preferences; initialFocusTab?: PreferencesTab }
```

Comment:
Since we call it the "selected" tab in the component, could we call this the `initialSelectedTab` instead?